### DISCUSSION TITLE
Node version incompatible

### DISCUSSION THREAD

**Discussion by sultandelux** [tailwindlabs/tailwindcss#5141] [Help] [answered]
error tailwindcss@2.2.6: The engine "node" is incompatible with this module. Expected version ">=12.13.0". Got "10.19.0"

Ubuntu

**Comment by dance2die** [comment, marked as answer]
The messages shows you have a version, 10.19.0
so you need to install a newer version of Node.js, greater than 12.13.0.  

https://tailwindcss.com/docs/upgrading-to-v2#upgrade-to-node-js-12-13-or-higher
